### PR TITLE
Ui fixes

### PR DESCRIPTION
**Pull request by freshface**
## Resolves the following issues

Some small UI enhancements

### CODE REVIEW

File: src/Backend/Modules/MediaLibrary/Layout/Templates/MediaBrowser.html.twig [hunk 1]
```diff
@@ -32,9 +32,9 @@
                     <div class="row">
                       <div class="col-sm-3">
                         <div id="mediaTree">
-                          <button class="btn btn-default btn-sm tree-collapsed" data-role="toggle-js-tree-collapse">
-                            <span data-role="toggle-js-tree-collapse-text">{{ 'lbl.OpenTreeNavigation'|trans }}</span>
-                          </button>
+                          <a class="tree-collapsed" data-role="toggle-js-tree-collapse">
```
Comment (carakas) [inline src/Backend/Modules/MediaLibrary/Layout/Templates/MediaBrowser.html.twig]: this is a button and not a link, it triggers an event and doesn't go to a link

File: src/Backend/Modules/MediaLibrary/Layout/Templates/MediaItemEdit.html.twig [hunk 2]
```diff
@@ -7,9 +7,9 @@
       <div class="alert alert-warning visible-xs visible-sm">
         <p>{{ macro.icon('exclamation-triangle') }}{{ 'msg.MoveMediaFoldersNotPossible'|trans|ucfirst }}</p>
       </div>
-      <button class="btn btn-default btn-sm tree-collapsed" data-role="toggle-js-tree-collapse">
-        <span data-role="toggle-js-tree-collapse-text">{{ 'lbl.OpenTreeNavigation'|trans }}</span>
-      </button>
+       <a class="tree-collapsed" data-role="toggle-js-tree-collapse">
```
Comment (carakas) [inline src/Backend/Modules/MediaLibrary/Layout/Templates/MediaItemEdit.html.twig]: idem

File: src/Backend/Modules/MediaLibrary/Layout/Templates/MediaItemIndex.html.twig [hunk 3]
```diff
@@ -7,9 +7,9 @@
       <div class="alert alert-warning visible-xs visible-sm">
         <p>{{ macro.icon('exclamation-triangle') }}{{ 'msg.MoveMediaFoldersNotPossible'|trans|ucfirst }}</p>
       </div>
-      <button class="btn btn-default btn-sm tree-collapsed" data-role="toggle-js-tree-collapse">
-        <span data-role="toggle-js-tree-collapse-text">{{ 'lbl.OpenTreeNavigation'|trans }}</span>
-      </button>
+       <a class="tree-collapsed" data-role="toggle-js-tree-collapse">
```
Comment (carakas) [inline src/Backend/Modules/MediaLibrary/Layout/Templates/MediaItemIndex.html.twig]: idem

File: src/Backend/Modules/MediaLibrary/Layout/Templates/MediaItemUpload.html.twig [hunk 4]
```diff
@@ -7,9 +7,9 @@
       <div class="alert alert-warning visible-xs visible-sm">
         <p>{{ macro.icon('exclamation-triangle') }}{{ 'msg.MoveMediaFoldersNotPossible'|trans|ucfirst }}</p>
       </div>
-      <button class="btn btn-default btn-sm tree-collapsed" data-role="toggle-js-tree-collapse">
-        <span data-role="toggle-js-tree-collapse-text">{{ 'lbl.OpenTreeNavigation'|trans }}</span>
-      </button>
+       <a class="tree-collapsed" data-role="toggle-js-tree-collapse">
```
Comment (carakas) [inline src/Backend/Modules/MediaLibrary/Layout/Templates/MediaItemUpload.html.twig]: idem

File: src/Backend/Modules/Pages/Layout/Templates/Add.html.twig [hunk 5]
```diff
@@ -7,9 +7,9 @@
       <div class="alert alert-warning visible-sm">
         <p>{{ macro.icon('exclamation-triangle') }}{{ 'msg.MovePagesNotPossible'|trans }}</p>
       </div>
-      <button class="btn btn-default btn-sm tree-collapsed" data-role="toggle-js-tree-collapse">
-        <span data-role="toggle-js-tree-collapse-text">{{ 'lbl.OpenTreeNavigation'|trans }}</span>
-      </button>
+       <a class="tree-collapsed" data-role="toggle-js-tree-collapse">
```
Comment (carakas) [inline src/Backend/Modules/Pages/Layout/Templates/Add.html.twig]: idem

File: src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig [hunk 6]
```diff
@@ -7,9 +7,9 @@
       <div class="alert alert-warning visible-sm">
         <p>{{ macro.icon('exclamation-triangle') }}{{ 'msg.MovePagesNotPossible'|trans|ucfirst }}</p>
       </div>
-      <button class="btn btn-default btn-sm tree-collapsed" data-role="toggle-js-tree-collapse">
-        <span data-role="toggle-js-tree-collapse-text">{{ 'lbl.OpenTreeNavigation'|trans }}</span>
-      </button>
+      <a class="tree-collapsed" data-role="toggle-js-tree-collapse">
```
Comment (carakas) [inline src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig]: idem

File: src/Backend/Modules/Pages/Layout/Templates/Index.html.twig [hunk 7]
```diff
@@ -7,9 +7,9 @@
       <div class="alert alert-warning visible-xs visible-sm">
         <p>{{ macro.icon('exclamation-triangle') }}{{ 'msg.MovePagesNotPossible'|trans|ucfirst }}</p>
       </div>
-      <button class="btn btn-default btn-sm tree-collapsed" data-role="toggle-js-tree-collapse">
-        <span data-role="toggle-js-tree-collapse-text">{{ 'lbl.OpenTreeNavigation'|trans }}</span>
-      </button>
+      <a class="tree-collapsed" data-role="toggle-js-tree-collapse">
```
Comment (carakas) [inline src/Backend/Modules/Pages/Layout/Templates/Index.html.twig]: idem

**Comment by carakas** [comment]
the only thing that remains is the alignment of the buttons to the left, is this something we will do @forkcms/frontenders

**Comment by mathiashelin** [comment]
You can remove the `btn-default` class if you use `btn-link`, see [documentation](https://getbootstrap.com/docs/3.3/css/#buttons).

File: src/Backend/Modules/MediaLibrary/Layout/Templates/MediaBrowser.html.twig [hunk 8]
```diff
@@ -32,8 +32,8 @@
                     <div class="row">
                       <div class="col-sm-3">
                         <div id="mediaTree">
-                          <button class="btn btn-default btn-sm tree-collapsed" data-role="toggle-js-tree-collapse">
-                            <span data-role="toggle-js-tree-collapse-text">{{ 'lbl.OpenTreeNavigation'|trans }}</span>
+                          <button class="btn btn-default btn-sm btn-link tree-collapsed" data-role="toggle-js-tree-collapse">
```
Comment (Katrienvh) [inline src/Backend/Modules/MediaLibrary/Layout/Templates/MediaBrowser.html.twig]: Do you have a screenshot of this? If i try this out the button background becomes white, that does not seem like an enchainment to me.

**Pull request closed without merging**